packages/react/src/components/Panel/Panel.base.tsx

```diff
@@ -227,7 +228,7 @@ export class PanelBase extends React.Component<IPanelProps, IPanelState> impleme
           ariaLabelledBy={this._headerTextId ? this._headerTextId : undefined}
           onDismiss={this.dismiss}
           className={_classNames.hiddenPanel}
-          {...popupProps}
+          {...popupPropsWithDefaults}
```

Comment:
If we do go ahead with this approach, it would be better to leave the props spread as it was and provide the default in the same way that other defaults are passed through.
```suggestion
          enableAriaHiddenSiblings={true}
          {...popupProps}
```